src/lib/es5.d.ts

```diff
@@ -1113,21 +1113,35 @@ interface JSON {
      * @param reviver A function that transforms the results. This function is called for each member of the object.
      * If a member contains nested objects, the nested objects are transformed before the parent object is.
      */
-    parse(text: string, reviver?: (this: any, key: string, value: any) => any): any;
+    parse(text: string, reviver?: (this: unknown, key: string, value: unknown) => unknown): any;
     /**
-     * Converts a JavaScript value to a JavaScript Object Notation (JSON) string.
-     * @param value A JavaScript value, usually an object or array, to be converted.
+     * Converts a JSON value to a JavaScript Object Notation (JSON) string.

```

Comment:
There's no such thing as a _JSON value_. JSON is a _notation_, or a text format. There's only _JSON-serializable value_.

Reply:
Sure, I'll change it to JSON-representable to match the official ECMAScript spec.